medicaltorch/datasets.py

```diff
@@ -143,11 +152,15 @@ def get_pair_data(self):
         for handle in self.input_handle:
             input_data.append(handle.get_fdata(cache_mode, dtype=np.float32))
 
+        gt_data = []
         # Handle unlabeled data
         if self.gt_handle is None:
             gt_data = None
-        else:
-            gt_data = self.gt_handle.get_fdata(cache_mode, dtype=np.float32)
+        for gt in self.gt_handle:
+            if gt is not None:
+                gt_data.append(gt.get_fdata(cache_mode, dtype=np.float32))
```

Comment:
same question than on `ivadomed` re data type.

Reply:
I have this weird error if I use np.uint8. I am not sure why. Should be investigated
![image](https://user-images.githubusercontent.com/49137243/78410780-6752f900-75db-11ea-83c2-01de14af70a3.png)